css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,99 @@
+@import "../page-vars";
+
+.ampstart-sidebar {
+  width: 80vw;
+  /* Keep the header bar visible when the sidebar is open */
+  height: calc(100vh - var(--layout-header-height-mobile));
```

Comment:
Tweaked the height of the sidebar as the design requires the header to be visible even when the sidebar is open